pkg/matcher/expr.go

```diff
@@ -0,0 +1,52 @@
+package matcher
+
+import "fmt"
+
+type (
+	// SimpleExpr is a simple expression to describe the condition:
+	//     (includes[0].Match(v) || includes[1].Match(v) || ...) && !(excludes[0].Match(v) || excludes[1].Match(v) || ...)
+	SimpleExpr struct {
+		Includes []string `yaml:"includes" json:"includes"`
+		Excludes []string `yaml:"excludes" json:"excludes"`
```

Comment:
i suggest to change to `Include` and `Exlcude`

Reply:
I use includes and excludes because test kitchen use this syntax:
<https://docs.chef.io/kitchen.html#syntax>

Reply:
ok, maybe my eglish is bad, this `s doesn't make sense to me in that context